internal/core/users/users.handler.go

```diff
@@ -101,23 +127,23 @@ func (h *handlerImpl) Register(c *fiber.Ctx) error {
 	return utils.ResponseMessage(c, http.StatusCreated, "User created")
 }
 
-// @router      /api/v1/user/ [put]
+// @router      /api/v1/user/me/personal-information [put]
 // @summary     Update current user personal information *use cookies*
 // @description Update specifying userId with formData **\***upload profile image in formData with field `profile_image`. Available formats are .png / .jpg / .jpeg
 // @tags        users
 // @produce     json
-// @param       formData formData models.UpdatingUserPersonalInfo true "User information"
-// @success     200	{object} models.Users
+// @param       formData formData models.UpdatingUserPersonalInfos true "User information"
+// @success     200	{object} models.Users "User personal information updated"
```

Comment:
This is a lost changed I've made last time. ฝ า ก change it to `models.MessageResponses`. tytytyty

Reply:
success 200 `models.Users` to `models.MessageResponses`